angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts

```diff
@@ -0,0 +1,683 @@
+            additionalIconClasses="sprk-c-Icon--l"
+            additionalClasses="sprk-u-Right--zero sprk-u-mrm"
+            screenReaderText="User Account"
+          >
+          </sprk-dropdown>
+        </li>
+      </ul>
+    </sprk-masthead>
+  `,
+  props: {
+    links: [
+      {
+        text: 'Navigation Item',
+        href: '/icons',
+        active: false
+      },
+      {
+        text: 'Your Item',
+        active: false,
+        subNav: [
+          {
+            text: 'Item 1',
+            href: '/links'
+          },
+          {
+            text: 'Item 2',
+            href: '#nogo'
+          },
+          {
+            text: 'Item 3',
+            href: '#nogo'
+          }
+        ]
+      },
+      {
+        text: 'Nav Item',
+        href: '#nogo',
+        active: false,
+        leadingIcon: 'settings'
+      },
+      {
+        text: 'Navigation Item',
+        href: '#nogo',
+        leadingIcon: 'chat-online',
+        subNav: [
+          {
+            text: 'Item 1',
+            href: '#nogo'
+          },
+          {
+            text: 'Item 2',
+            href: '#nogo'
+          },
+          {
+            text: 'Item 3',
+            href: '#nogo'
+          }
+        ]
+      },
+      {
+        text: 'Item',
+        href: '#nogo',
+        active: false,
+        leadingIcon: 'chat-online'
+      }
+    ],
+    extendedNarrowLinks: [
+      {
+        text: 'Item 1',
+        href: '#nogo',
+        subNav: [
+          {
+            text: 'Item 1',
+            href: '/links'
+          },
+          {
+            text: 'Item 2',
+            href: '/links'
+          },
+          {
+            text: 'Item 3',
+            href: '/links'
+          }
+        ]
+      },
+      {
+        text: 'Item 2',
+        href: '#nogo',
+        active: true
+      },
+      {
+        text: 'Item 3',
+        href: '#nogo',
+        subNav: [
+          {
+            text: 'Item 1',
+            href: '/links'
+          },
+          {
+            text: 'Item 2',
+            href: '/links'
+          },
+          {
+            text: 'Item 3',
+            href: '/links'
+          }
+        ]
+      },
+      {
+        text: 'Item 4',
+        href: '#nogo'
+      },
+      {
+        text: 'Item 5',
+        href: '#nogo'
+      },
+      {
+        text: '(586) 123-4567',
+        href: '#nogo',
+        active: false,
+        leadingIcon: 'landline'
+      },
+      {
+        text: 'Talk To Us',
+        href: '#nogo',
+        active: false,
+        leadingIcon: 'call-team-member'
+      },
+      {
+        text: 'Settings',
+        href: '#nogo',
+        active: false,
+        leadingIcon: 'settings'
+      },
+      {
+        text: 'My Account',
+        href: '#nogo',
+        active: false,
+        leadingIcon: 'user',
+        subNav: [
+          {
+            text: 'Change Username',
+            href: '/links'
```

Comment:
What happens when this link with the value of '/links' is clicked in storybook? I don't think we have a route there.

Reply:
It doesn't do anything or navigate anywhere. But I can clean these up to be #nogo if that's preferable.